edge-apps/clock/static/css/styles.css

```diff
@@ -0,0 +1,918 @@
+    padding: 0rem;
+    box-sizing: border-box;
+}
+
+html {
+    font-size: 14px;
+}
+
+:root {
+    --theme-color-primary: #972EFF;
+    --theme-color-secondary: #ADAFBE;
+    --theme--color-tertiary: #FFFFFF;
+    --theme-color-background: #C9CDD0;
+}
+
+
+body {
+    height: 100dvh;
+    font-family: 'Aeonik', sans-serif;
+}
+
+
+.main-container {
+    display: flex;
+    flex-direction: row;
+    gap: 2rem;
+    height: 100%;
+    padding: 2rem;
+    /* height: 100dvh; */
+    background-color: var(--theme-color-background);
+}
+
+
+.primary-card {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 50%;
+    height: 100%;
+    border-radius: 3rem;
+    gap: 2rem;
+    line-height: 1;
+    background-color: var(--theme-color-primary);
+    color: var(--theme--color-tertiary);
+}
+
+
+.primary-card-clock {
+    line-height: 1;
+    font-size: 8rem;
+    color: var(--theme--color-tertiary);
+}
+
+
+.secondary-container {
+    width: 50%;
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+    justify-content: space-between;
+}
+
+
+.row-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    gap: 2rem;
+    height: 50%;
+}
+
+
+.secondary-card {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: auto;
+    gap: 2rem;
+    border-radius: 3.481rem;
+    background-color: var(--theme--color-tertiary);
+}
+
+
+.secondary-card-number-container {
+    display: flex;
+    flex-direction: row;
+    gap: 3rem;
+    align-items: baseline;
+}
+
+.secondary-card-time-am-pm {
+    font-size: 4rem;
+    color: var(--theme-color-secondary);
+}
+
+.secondary-card-number {
+    font-size: 14.5rem;
+    color: var(--theme-color-primary);
+}
+
+.secondary-card-label {
+    font-size: 3rem;
+    color: var(--theme-color-secondary);
+}
+
+.brand-logo {
+    width: 12rem;
+}
+
+
+.info-text {
+    font-size: 1.75rem;
+    color: var(--theme-color-primary);
+}
+
+
+.info-card {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 2rem;
+
+}
+
+.date-card {
+    gap: 0.25rem;
+}
+
+
+.date-number {
+    line-height: 1;
+    font-size: 10rem;
+    color: var(--theme-color-primary);
+}
+
+.date-text {
+    font-size: 5rem;
+    color: var(--theme-color-secondary);
+}
+
+
+/* ---------Clock Start ------------ */
+
+.clock-container {
+    width: 36.43rem; /* 510px / 14 */
+    height: 36.43rem; /* 510px / 14 */
+    background: #ffffff;
+    border-radius: 50%;
+    display: flex;
+    justify-self: center;
+    align-items: center;
+}
+
+.clock-container .clock {
+    width: 30rem; /* 420px / 14 */
+    height: 30rem; /* 420px / 14 */
+    border-radius: 50%;
+    margin: auto;
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.clock .number_hours,
+.clock .seconds_bar {
```

Comment:
Follow the same convention for naming, preferably hyphen case numbers-hours, seconds-bar

Reply:
Fixed in https://github.com/Screenly/Playground/pull/192/commits/477da927959743a3b653b30994ae3eb0b675542d